packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -543,20 +507,19 @@ export const makeInnerVault = (
     const giveColl = proposal.give.Collateral || emptyCollateral;
     const wantColl = proposal.want.Collateral || emptyCollateral;
 
-    const newCollateralPre = applyDelta(collateralPre, giveColl, wantColl);
+    const newCollateralPre = addSubtract(collateralPre, giveColl, wantColl);
     // max debt supported by current Collateral as modified by proposal
-    const maxDebtPre = await maxDebtFor(newCollateralPre);
+    const maxDebtPre = await manager.maxDebtFor(newCollateralPre);
     assert(
       updaterPre === state.outerUpdater,
       X`Transfer during vault adjustment`,
     );
     assertActive();
 
-    // After the AWAIT, we retrieve the vault's allocations again.
-    // Get new balances after calling the priceAuthority, so we can compare
-    // to the debt limit based on the new values.
+    // After the AWAIT, we retrieve the vault's allocations again,
```

Comment:
Why is `AWAIT` all caps? Would this suffice?
```suggestion
    // After the `await`, we retrieve the vault's allocations again,
```

Reply:
Its because we used to write a `// AWAIT` comment before we came up with our current rule.